projects/media-viewer/src/lib/annotations/annotation.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="annotation-wrapper">
```

Comment:
I'm confused, has this file been moved or something?

Reply:
renamed